public/app/features/dashboard/components/ShareModal/SharePublicDashboard/SharePublicDashboardUtils.ts

```diff
@@ -77,7 +77,7 @@ export const generatePublicDashboardUrl = (accessToken: string): string => {
 };
 
 export const generatePublicDashboardConfigUrl = (dashboardUid: string): string => {
-  return `/d/${dashboardUid}?shareView=public-dashboard`;
+  return `/d/${dashboardUid}?shareView=public_dashboard`;
```

Comment:
Can we use the new enum instead of a hardcoded string to avoid this happening again?

Reply:
yes, you're right.